compiler/src/iree/compiler/DispatchCreation/FuseEncodingOpsIntoDispatchRegions.cpp

```diff
@@ -48,15 +48,9 @@ static bool isFusableWithSetEncoding(Operation *op) {
     if (llvm::none_of(op.getResultTypes(), llvm::IsaPred<ShapedType>)) {
       continue;
     }
-    if (isa<tensor::CollapseShapeOp, tensor::ExpandShapeOp, tensor::EmptyOp>(
-            op)) {
-      continue;
-    }
-    auto linalgOp = dyn_cast<linalg::LinalgOp>(op);
-    if (!linalgOp) {
-      return false;
-    }
-    if (linalgOp.getNumReductionLoops() != 0) {
+    if (!isa<tensor::CollapseShapeOp, tensor::ExpandShapeOp, tensor::EmptyOp,
+             IREE::Encoding::SetEncodingOp, IREE::Encoding::UnsetEncodingOp,
+             linalg::LinalgOp, IREE::LinalgExt::AttentionOp>(op)) {
```

Comment:
Do we need the method now? Are we being conservative that avoids the fusion for other linalg ext ops?

It is okay to keep the list, I'm just curious about this.